.drone.starlark

```diff
@@ -19,7 +19,7 @@ config = {
 		},
 	},
 
-	'javascript': True,
+	'javascript': False,
```

Comment:
Note: before this PR there was no starlark code that executed JS tests, so having this set to True had no effect.

I have set it to false for now, because the JS tests have not yet been looked at to run on drone. That is a separate job.